packages/cli/src/util/build/import-builders.ts

```diff
@@ -201,15 +205,46 @@ async function installBuilders(
     if (err.code !== 'EEXIST') throw err;
   }
 
-  output.debug(`Installing Builders: ${Array.from(buildersToAdd).join(', ')}`);
-  await spawnAsync('yarn', ['add', '@vercel/build-utils', ...buildersToAdd], {
-    cwd: buildersDir,
-  });
+  output.log(`Installing Builders: ${Array.from(buildersToAdd).join(', ')}`);
+  try {
+    await spawnAsync('yarn', ['add', '@vercel/build-utils', ...buildersToAdd], {
+      cwd: buildersDir,
+      stdio: 'pipe',
+    });
+  } catch (err: unknown) {
+    if (isError(err)) {
+      if (isErrnoException(err) && err.code === 'ENOENT') {
+        // `yarn` is not installed
+        err.message = `Please install ${cmd('yarn')} before continuing`;
+        (err as any).link = 'https://classic.yarnpkg.com/lang/en/docs/install';
+      } else {
+        const message = errorToString(err);
+        const notFound = /"(.*): Not found"/.exec(message);
+        (err as any).link = 'https://vercel.link/npm-install-failed-dev';
```

Comment:
This redirect doesnt go to the correct section

Reply:
It does, but we should use a different link here to be more generic. That's still TODO here.

Reply:
Using new link (https://vercel.link/builder-dependencies-install-failed) that points to updated docs section.